java/ql/src/Security/CWE/CWE-1004/InsecureTomcatConfig.ql

```diff
@@ -0,0 +1,26 @@
+/**
+ * @name Disabled Tomcat HTTPOnly config for XSS protection
+ * @description Disabing HTTPOnly can cause session cookies leaked by XSS attack.
```

Comment:
Thanks for following our conventions for the description. A small suggestion to make it clearer:

```suggestion
 * @description Disabling 'HttpOnly' leaves session cookies vulnerable to an XSS attack.
```